app/Console/Commands/BulkReviewPosts.php

```diff
@@ -57,20 +57,16 @@ public function handle()
     {
         info('rogue:bulkreviewposts: Starting to bulk review posts!');
 
-        $postType = $this->option('type') ?? null;
         $logfreq = $this->option('logfreq');
         $log = $this->option('log');
         $tags = $this->option('tag') ?? null;
 
         $postQuery = Post::where([
             ['campaign_id', $this->argument('campaign')],
             ['status', $this->argument('oldStatus')],
+            ['type', $this->argument('type')],
         ]);
```

Comment:
you should be able to just do `->get()` here instead of in a separate statement. 
```
         $postQuery = Post::where([
             ['campaign_id', $this->argument('campaign')],
             ['status', $this->argument('oldStatus')],
            ['type', $this->argument('type')],
         ])->get();

```

Reply:
oh right! Updated :)